solutions/swb-reference/integration-tests/tests/multiStep/dataset.test.ts

```diff
@@ -132,6 +133,28 @@ describe('multiStep dataset integration test', () => {
       .dataset(dataSet.id)
       .associateWithProject(unassociatedProject.id, 'read-only');
 
+    console.log('CHECK PROJECT PERMISSIONS FOR DATASET');
+    const { data: responseData } = await adminSession.resources.datasets
+      .dataset(dataSet.id)
+      .listAccessPermissions();
+    const sortedActual: DataSetPermission[] = responseData.data.permissions.sort(
```

Comment:
we love a custom sort method!

Reply:
i bet theres some backstory here i don't know about haha

Reply:
relying on default sort methods result in very frustrating bugs if the default does not behave how you think it behaves! We faced a bug of this nature for public preview and it was harrowing (okay, I'm being a tad dramatic).